Automattic/vip-go-mu-plugins
# Search: Add mapping settings for slow logs

## Description
This adds the post mapping settings for slow logs. Will be only effective on new indexes or when the index is setup again.

## Changelog Description
### Plugin Updated: Enterprise Search

Add slow log settings to post mapping.
## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Run `wp vip-search put-mapping`
2) Do a GET request on http://ES_URL_HERE/vip-200508-post-1/_settings and see the below bits returned from the settings:

```
               "search": {
                    "slowlog": {
                        "threshold": {
                            "fetch": {
                                "warn": "800ms"
                            },
                            "query": {
                                "warn": "1900ms",
                                "info": "1s"
                            }
                        }
                    }
                },
```

```
                "indexing": {
                    "slowlog": {
                        "threshold": {
                            "index": {
                                "warn": "4900ms"
                            }
                        },
                        "source": "1000"
                    }
                },
```
